src/conversion_helper.js

```diff
@@ -758,34 +780,66 @@ function filterConversions(potentialConversions) {
 /*
   Input: imperial conversions
     [
-      { "imperial" : { "number" : 10000, "unit" : " miles" } },
-      { "imperial" : { "number" : 30, "unit" : " mpg" } }
+      { "imperial" : { "numbers" : 10000, "unit" : " miles" } },
+      { "imperial" : { "numbers" : 30, "unit" : " mpg" } }
     ]
   Output: metric and imperial conversions
     [
       { 
         "imperial" : 
-          { "number" : 10000, "unit" : " miles" }, 
+          { "numbers" : 10000, "unit" : " miles" }, 
         "metric": 
-          { "number" : 16093.44, "unit" : " km" } 
+          { "numbers" : 16093.44, "unit" : " km" } 
       },
       { 
         "imperial" : 
-          { "number" : 30, "unit" : " mpg" }, 
+          { "numbers" : 30, "unit" : " mpg" }, 
         "metric" : [
-          { "number" : 12.7543, "unit" : " km/L" },
-          { "number" : 7.84049, "unit" : " L/100km" } 
+          { "numbers" : 12.7543, "unit" : " km/L" },
+          { "numbers" : 7.84049, "unit" : " L/100km" } 
```

Comment:
Should the numbers in this documentation be changed to an array as well, like [7.84049]?

Aside: Things like this is why I'm not a big fan of using comments for documentation, it's so easy for it to get out-of-date. Ideally the code would be clear enough to easily figure out inputs/outputs but unfortunately we're working with javascript here and it's impossible* to tell what kind of parameters functions expect. Perhaps we can use tests as our main documentation source... something to think about for the future.

*impossible for me at least. If someone knows a trick, please share

Reply:
Yeah, I'll do that.